drivers/mpsl/clock_control/Kconfig

```diff
@@ -24,6 +24,14 @@ config CLOCK_CONTROL_NRF_K32SRC_XTAL
 config CLOCK_CONTROL_NRF_K32SRC_SYNTH
 	bool "Synthesized from HFCLK"
 
+config CLOCK_CONTROL_NRF_K32SRC_EXT_LOW_SWING
+	depends on SOC_SERIES_NRF52X
+	bool "External low swing"
+
+config CLOCK_CONTROL_NRF_K32SRC_EXT_FULL_SWING
+	depends on SOC_SERIES_NRF52X
+	bool "External full swing"
```

Comment:
is this descriptive enough for our users to know what it means ?
Also, please add a help text with a more descriptive text field.

Reply:
@tejlmand , see https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1974#discussion_r387666006